### PR TITLE
Director update to role.md

### PR DESCRIPTION
**Pull request by its-a-lisa-at-work**
Made some content changes to role.md

Still need to check a few links and run prettier on it so it is not complete or ready for review yet

### CODE REVIEW

File: how_we_work/roles.md [hunk 1]
```diff
@@ -1,66 +1,74 @@
...
 - Ensuring the team is supported by other leadership
 - Ensuring the team fulfills its various [responsibilities](https://handbook.tts.gsa.gov/tech-portfolio/), including meeting deadlines
 - Resolving conflicts within the team, including being a tie-breaker when needed
 
-## Original Job Announcements
+### Supporting Documentation
 
-- [Tech Portfolio Specialist](https://join.tts.gsa.gov/join/technology-portfolio-specialist/) x 2
 - [Technology Director](https://join.tts.gsa.gov/join/technology-portfolio-director/)
+- [TTS Tech Portfolio: Leadership Roles & Responsibilities - Director](https://docs.google.com/document/d/1B4rtZd06w7ITABrjrGWRjAfU4f-go2jnuO_D0PokJMw/edit#heading=h.5lx1f0htbp8v)
```
Comment (afeld) [inline how_we_work/roles.md:45]: Not a blocker here, but would be good to merge that content from the doc into the section above so it's a) in one place, and b) public.

Comment (its-a-lisa-at-work) [reply how_we_work/roles.md:45]: I agree that its good to have it in one place; I just think hat one place should be the doc and not this repo. 

I don't think this is the type of thing that needs to be public; I don't think it is necessarily bad to have it public, but I think the way it is now is functional and readable and the pertinent information is accessible and the people that it applies to are able to see the details.

Comment (its-a-lisa-at-work) [reply how_we_work/roles.md:42]: @afeld we kind of talked about this at the retro, but don't feel like we resolved it -- should we create an issue for this specifically to talk through? Do we need to do that before resolving this?

Comment (its-a-lisa-at-work) [reply how_we_work/roles.md:42]: @JJediny could use your weigh in here as well

**Comment by its-a-lisa-at-work** [comment]
I'm going to merge this pull request and create a new issue related to having consensus on what our "canonical" source should be

**Pull request merged**